plugins/commands/plugins/src/main/kotlin/PluginsCommand.kt

```diff
@@ -62,22 +62,17 @@ class PluginsCommand(descriptor: PluginDescriptor = PluginsCommandFactory.descri
         echo()
 
         plugins.forEach { plugin ->
-            echo(
-                HorizontalRule(
-                    buildString {
-                        append(plugin.displayName)
-                        if (plugin.id != plugin.displayName) append(" (id: ${plugin.id})")
-                    },
-                    "-"
-                )
-            )
+            echo(HorizontalRule(plugin.displayName))
```

Comment:
If the goal is to check if code changes affect the plugin IDs, maybe adding a `--list-ids` option or so to the command to just list the plugin IDs would be more helpful?

Reply:
I was indeed considering that, but then decided that adjusting the current output is less work, and the use-case for a specialized `--list-ids` to be a bit limited.